core/src/db/CMakeLists.txt

```diff
@@ -116,7 +116,7 @@ if ( MILVS_WITH_AWS )
 endif ()
 
 if ( NOT ${CMAKE_SYSTEM_PROCESSOR} MATCHES "aarch64" )
-    list( APPEND THIRD_PARTY_LIBS libquadmath.a )
+    list( APPEND THIRD_PARTY_LIBS quadmath )
```

Comment:
libquadmath is under the license LGPL v2, it's correct to modify it to dynamic link.

Reply:
Fixed